Makefile

```diff
@@ -120,11 +133,14 @@ CFLAGS := -Wall \
           -Wno-unknown-pragmas \
           -Wundef \
           -Wconversion \
-          -Werror \
-          $(DEVICE_INFO) \
+	  $(DEVICE_INFO) \
           $(DEBUG_FLAGS) \
           $(MEMORY_CHECK_CFLAGS) \
-          -O3
+          -O0	#This option for optimization is for preventing variables that
+		#store register values from being "optimized out".
+		#Might significantly degrade the FTL's performance,
+		#and shall return to -O3 after fixing the up-mentioned problems
+		#with "volatile", etc..
```

Comment:
Please don't change this part. I suggest overriding CFLAGS in the CHIP2CHIP_DEVICE branch.

Reply:
https://github.com/BlaCkinkGJ/Flash-Translation-Layer/pull/43/files#r1206518154